bin/start

```diff
@@ -34,4 +34,6 @@ $PROJECT_ROOT_DIR/bin/wait-for-postgres $PROJECT_ROOT_DIR/bin/migrate-db
 
 docker stack deploy -c $API_SERVER_STACK_FILE $STACK_NAME
 
+${BASH_SOURCE%/*}/start-ipfs
+
```

Comment:
Other script invocations in the same script use `$PROJECT_ROOT_DIR/bin/`, perhaps this one should as well - for consistency and bevcause PROJECT_ROOT_DIR is clearer than ${BASH_SOURCE%/*}

Reply:
Agree, will fix